### ISSUE TITLE
fieldtrip format required?

### ISSUE THREAD

**Issue by mariannalanza** [fahrenfort/ADAM#159]
I started testing your scripts on my data (.set already epoched, 2 classes). It keeps giving me the error: "Error using read_raw_data('The dimord field seems missing and/or dimensions in the dataset seem to be off. When you are using fieldtrip format, make sure the input files have dimord rpt_chan_time (trial x channel x time).')". I'm not using fieldtrip format. This occurs both using cfg.raw_or_tfr = 'tfr';  or 'raw'. So, I downloaded your data from https://osf.io/eh4u8 and tested on the script made available https://osf.io/chz37 and the very same error occurred. I'm currently using Matlab R2022b.  What I'm missing?

**Comment by fahrenfort** [comment]
Hi, the most likely reason is that you're using the latest version of fieldtrip. The toolbox doesn't work with the latest version, I still need to fix that but have not had time yet. Also see https://github.com/fahrenfort/ADAM, under “Requirements”. Hope this helps, cheers,

Johannes

Op 12 apr 2024 om 22:31 heeft mariannalanza ***@***.***> het volgende geschreven:

﻿


I started testing your scripts on my data (.set already epoched, 2 classes). It keeps giving me the error: "error('The dimord field seems missing and/or dimensions in the dataset seem to be off. When you are using fieldtrip format, make sure the input files have dimord rpt_chan_time (trial x channel x time).')". I'm not using fieldtrip format. This occurs both using cfg.raw_or_tfr = 'tfr'; or 'raw'. So, I downloaded your data from https://osf.io/eh4u8 and tested on the script made available https://osf.io/chz37 and the very same error occurred. I'm currently using Matlab R2022b. What I'm missing?
